### DISCUSSION TITLE
Dice Pools & Modifiers

### DISCUSSION THREAD

**Discussion by graylion-wa** [mDehning/wa-dice-roll#14] [Q&A]
I did think of another question that is formula driven i think. In counting successes modifiers are applied as additional successes applying to the final success count and not related to any single die.

Using the last example 

**5d10 >=7 !>=9**

Original Roll: 8, 3, 7, 9, 2 (3 successes)
Reroll 1: 10 (1 success)
Reroll 2: 8 (1 success)
Successes: 5

And adding a modifier ...

5d10 >=7 !>=9 +1

Original Roll: 8, 3, 7, 9, 2 (3 successes)
Reroll 1: 10 (1 success)
Reroll 2: 8 (1 success)
Modifier: 1
Successes: 6

For these questions I'm sticking to actual systems without house-rules or variations.

Systems References
* 2d20 Modiphius 
* World of Darkness 
* Chronicles of Darkness
* Storypath
* Exalted

**Comment by mDehning** [comment]
That is, how the modificators currently work. They will add their number to the number of successes (or to the sum of all dice) after all the rolls have been done.

What is the question?

**Reply by graylion-wa** [reply]
Your answer was perfect when counting successes modifiers add to successes.

Thank you
